stdlib/public/Concurrency/TaskStatus.cpp

```diff
@@ -433,7 +442,44 @@ static void performCancellationAction(TaskStatusRecord *record) {
   // FIXME: allow dynamic extension/correction?
 }
 
+/// Perform any cancellation actions required by the given record.
+static void performGroupCancellationAction(TaskStatusRecord *record,
```

Comment:
invoked when we need to cancel "all children of this group, that this task is currently running"